### PR TITLE
Change string to bool

### PR DESCRIPTION
**Pull request by jahbenjah**
Change string to bool because the string type is not a primitive type

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by mairaw** [comment]
Build failed. Trying again.

File: docs/core/about.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ C#, Visual Basic, and F# languages can be used to write applications and librari
 
 .NET Core exposes APIs for many scenarios, a few of which follow:
 
-- Primitive types, such as [string][string] and [int][int].
+- Primitive types, such as [bool][bool] and [int][int].
```
Comment (rpetrusha) [inline docs/core/about.md]: For the link to be resolved, line 36 has to be modified as well. It should be:
[bool]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/bool

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for making this correction, @jahbenjah. One further change is needed, though, so that the link to the bool article can be resolved.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @jahbenjah, for making the additional change. We'll merge your PR once we've updated our live branch. Your changes should appear live on docs.microsoft.com in the next day or two.

**Pull request merged**